src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -721,5 +723,37 @@ private static bool GetSupportsSha3()
 
             return false;
         }
+
+        private static bool DetermineBinaryFormatterSupport()
+        {
+            Assembly assembly = typeof(System.Runtime.Serialization.Formatters.Binary.BinaryFormatter).Assembly;
+            AssemblyName name = assembly.GetName();
+            Version assemblyVersion = name.Version;
+
+            if (IsNetFramework)
+            {
+                return true;
+            }
+
+            if (IsNativeAot)
+            {
+                Console.WriteLine("BinaryFormatter is disabled in NativeAOT");
```

Comment:
Did you intend to leave these Console.WriteLines in?

Reply:
I do/did.  When a test wants to use BinaryFormatter it leaves a place where it becomes clear why it's there or not.

But, if there's objection, I can certainly remove them.